test/integration/parachain/staging/sequential/replace.test.ts

```diff
@@ -123,11 +123,13 @@ describe("replace", () => {
                     // eslint-disable-next-line no-constant-condition
                     while (true) {
                         const header = await api.rpc.chain.getHeader(hash);
-                        try {
-                            hash = await api.rpc.chain.getBlockHash(header.number.toNumber() + 1);
-                        } catch (_) {
-                            sleep(SLEEP_TIME_MS);
+                        const nextHash = await api.rpc.chain.getBlockHash(header.number.toNumber() + 1);
```

Comment:
This test had to be refactored because it started to fail: `api.rpc.chain.getBlockHash` started to return 0 instead if throwing when provided block height is higher than current block.